rero_ils/config.py

```diff
@@ -3261,3 +3274,36 @@ def _(x):
 RERO_ILS_LANGUAGE_MAPPING = {
     'dum': 'dut'  # neerlandais
 }
+
+# EXPORT MAPPING
+# ================
+RERO_ILS_EXPORT_MAPPER = {
+    'ris': {
+        'doctype_mapping': {
+            'BOOK': lambda m, s: (m == 'docmaintype_book'
+                                  and s not in ['docsubtype_manuscript',
+                                                'docsubtype_thesis',
+                                                'docsubtype_e-book']
+                                  ) or m in ['docmaintype_children',
+                                             'docmaintype_comic',
+                                             'docmaintype_leaf'],
+            'ART': lambda m, s: m == 'docmaintype_image'
+                                or s == 'docsubtype_kamishibai',
+            'JOUR': lambda m, s: m == 'docmaintype_article',
+            'MUSIC': lambda m, s: m == 'docmaintype_audio'
+                                  and s == 'docsubtype_music',
+            'EBOOK': lambda m, s: m == 'docmaintype_book'
+                                  and s == 'docsubtype_e-book',
+            'MANSCPT': lambda m, s: m == 'docmaintype_book'
+                                    and s == 'docsubtype_manuscript',
+            'THES': lambda m, s: m == 'docmaintype_book'
+                                 and s == 'docsubtype_thesis',
+            'SOUND': lambda m, s: m == 'docmaintype_audio'
+                                  and not s == 'docsubtype_music',
+            'MAP': lambda m, s: m == 'docmaintype_map',
+            'VIDEO': lambda m, s: m == 'docmaintype_movie_series',
+            'JFULL': lambda m, s: m == 'docmaintype_serial',
+            'SER': lambda m, s: m == 'docmaintype_series',
+        },
+    }
+}
```

Comment:
Is this the best place to place this configuration ? My feeling is this file should contains the application configuration. Maybe it could be placed into a `rero_ils.modules.documents.config.py` file. Not blocking, just an idea to think about.

An other consideration should be to use constants (from document.models.py) instead of string for comparaison.
This isn't blocking too.